apps/hpc-cdm/src/app/components/enketo/overrides/dialog.ts

```diff
@@ -11,6 +11,7 @@ type DialogContentObject = {
 };
 
 // TODO: implement prompts at some point
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const alert = (content: string | DialogContentObject) => Promise.reject('TODO');
```

Comment:
These are resolved by putting an underscore in front of a variable. But, in this case, I'd just remove it and `prompt` below, as they aren't used anywhere and this TODO will likely never be TODONE
```suggestion
const alert = (_content: string | DialogContentObject) => Promise.reject('TODO');
```